recipes/adios-python/meta.yaml

```diff
@@ -0,0 +1,61 @@
+{% set name = "adios-python" %}
+{% set version = "1.13.1" %}
+{% set sha256 = "684096cd7e5a7f6b8859601d4daeb1dfaa416dfc2d9d529158a62df6c5bcd7a0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: http://users.nccs.gov/%7Epnorbert/adios-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [win]
+
+requirements:
+  host:
```

Comment:
Don't you need to add a build section with compilers? It seems like you are building C/C++ code through cython